sql/search_aggregates_v5.sql

```diff
@@ -0,0 +1,91 @@
+WITH
+  augmented AS (
+  SELECT
+    s.*,
+    sc.element.source AS source,
+    sc.element.count AS count,
+    sc.element.engine AS engine,
+    (
+    SELECT
+      ANY_VALUE(element.version)
+    FROM
+      UNNEST(active_addons.list)
+    WHERE
+      element.addon_id = 'followonsearch@mozilla.com') AS addon_version,
```

Comment:
This should use `udf_mode_last` instead of `ANY_VALUE`, so that the result is reproducible:

```diff
-    (
+    udf_mode_last(ARRAY(
     SELECT
-      ANY_VALUE(element.version)
+      element.version
     FROM
       UNNEST(active_addons.list)
     WHERE
-      element.addon_id = 'followonsearch@mozilla.com') AS addon_version,
+      element.addon_id = 'followonsearch@mozilla.com')) AS addon_version,
```

(and prepend the function from `udf/udf_mode_last.sql`)